src/db/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I'm not sure we can have two `package.jsons` like that. This calls for either a monorepo, or an external package.

Reply:
"Can have" in what sense?